rest-server/src/components/mapbox/mapboxRouter.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import { fetchUserLocationCoordinates } from './mapboxControllers';
+
+const router = express.Router();
+
+router.route('/:_id')
```

Comment:
I treated the mapbox external API call as a separate route from the user route because it went beyond just accessing the data from mongoDB. I figured since it's also making a request to the mapbox API, I'd leave it out of the basic CRUD function routes to avoid clutter. I can fix this.

Reply:
Nothing you need to do here, I just wanted to understand your thinking.  Thank you.

Reply:
Generally, would you consider it to be better practice keeping the mapbox API route as an endpoint on the users route because it is still related to the user model?